backend/main.py

```diff
@@ -0,0 +1,99 @@
+import sqlite3, uuid, json
+from bottle import Bottle, run, request, response
+
+app = Bottle()
+
+# DB mit SQLite
+con = sqlite3.connect("users.db")
+cur = con.cursor()
+
+def init():
+    createDB()
+
+def createDB():
+    sql_query = """
+    CREATE TABLE IF NOT EXISTS users (
+    id VARCHAR(36),
```

Comment:
This should probably be marked as the primary key.

Reply:
Done.